docs/getting-started.md

```diff
@@ -3,7 +3,7 @@
 ## Quick Start - Unpack a firmware file and display it in the GUI
 
 !!! warning
-    OFRAK is a Python library supporting Python3.7 and up. First and foremost, make sure your Python and pip installations are for Python3.7+! Python 3.8 is recommended, as this is the version we primarily test OFRAK with.
+    OFRAK is a Python library supporting Python3.8 and up. First and foremost, make sure your Python and pip installations are for Python3.8+! Python 3.8 is recommended, as this is the version we primarily test OFRAK with.
```

Comment:
Hm. Technically OFRAK still supports Python3.7; it is just `ofrak-angr` that requires 3.8 and up.

Are you suggesting that we drop 3.7 support with this commit? @rbs-jacob, would appreciate your thoughts on this. If we decide to do so, we should update all of the CHANGELOGs (and probably the "python_requires") in this repo.